release.sh

```diff
@@ -98,56 +103,86 @@ git checkout "$BRANCH"
 git pull
 git checkout --detach
 
-# Edit package.json and bower.json to the right version (see
+# Edit package.json to the right version (see
 # http://stackoverflow.com/a/22084103 for why we need the .bak file to make
 # this mac & linux compatible)
 sed -i.bak -E 's|"version": "[^"]+",|"version": "'$VERSION'",|' package.json
 rm -f package.json.bak
 
-# Build generated files and add them to the repository (for bower)
+# Build generated files and add them to the repository
 git clean -fdx dist
 yarn dist
 sed -i.bak -E '/^\/dist\/$/d' .gitignore
 rm -f .gitignore.bak
 git add .gitignore dist/
 
-# Update the version number in CDN URLs included in the README files,
+# Edit docs to use CSS from CDN (#1520)
+# grep -l '{@stylesheet: static/katex.min.css}' docs/*.md | xargs sed -i.bak \
+#     's|{@stylesheet: static/katex.min.css}|<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@./dist/katex.min.css" integrity="sha256-katex.min.css" crossorigin="anonymous"/>|'
+
+# Update the version number in CDN URLs included in the README and the documentation,
 # and regenerate the Subresource Integrity hash for these files.
-node update-sri.js "${VERSION}" README.md contrib/*/README.md dist/README.md
+node update-sri.js "${VERSION}" README.md contrib/*/README.md dist/README.md \
+    docs/*.md docs/*.md.bak website/pages/index.html
+
+# Generate a new version of the docs and publish the website
+pushd website
+npm run version "${VERSION}"
+
+# Restore docs to use local built CSS
+# for file in ../docs/*.md.bak; do
+#     mv -f "$file" "${file%.bak}"
+# done
+
+USE_SSH=true npm run publish-gh-pages
+popd
 
 # Make the commit and tag, and push them.
-git add package.json bower.json README.md contrib/*/README.md dist/README.md
+git add package.json README.md contrib/*/README.md dist/README.md \
+    docs/*.md website/
```

Comment:
I'm a little confused why we're adding `website` to the tag.  Should this just be `website/pages/index.html` from the `update-sri.js` command above?

Reply:
@edemaine `yarn version` command copies current `docs` and sidebar into `website/versioned_docs` and `website/versioned_sidebars`, respectively, and edits `versions.json`.

Reply:
It *might* be preferable to more explicitly add the things you want (although there are other checks for uncommitted changes, I guess).